### PR TITLE
Machine Learning

### PR DESCRIPTION
**Pull request by Franco98404**
Copyright corrections, implementation of an abstract class, improvement in the performance, enhancements in the return of the predictions, and other changes.

### CODE REVIEW

File: MACHINE_LEARNING/machine_learning/models/model.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
+from abc import ABC, abstractmethod
```
Comment (lcarrietas) [inline MACHINE_LEARNING/machine_learning/models/model.py:12]: please add the copyright information and class description

Comment (Franco98404) [reply MACHINE_LEARNING/machine_learning/models/model.py:12]: Thank you!

**Review by lcarrietas — APPROVED** [review]
Good!

**Review by MatiasBonifacio — APPROVED** [review]
It's okay but in the next one don't leave the function "def __init__(self): pass" because it takes up space in memory.

**Pull request merged**